specifications/android.md

```diff
@@ -49,18 +49,19 @@ The exposed broadcast receiver of the distributor application MUST handle 2 diff
 
 ### org.unifiedpush.android.distributor.REGISTER
 
-The connector send this action to register to push messages. The intent MUST contain 2 String extras:
-* application: with the end user application package name. The distributor MUST be able to handle many connections with a single application.
-* token: with a random token to identify the connection between the connector and the distributor. It MUST be unique on distributor side.
+The connector send this action to register to push messages. The intent MUST contain 3 extras:
+* application (String): with the end user application package name. The distributor MUST be able to handle many connections with a single application.
+* token (String): with a random token to identify the connection between the connector and the distributor. It MUST be unique on distributor side.
+* versions (Array\<Int\>): an array of the major versions number supported by the connector.
```

Comment:
Could we check for the MESSAGEv2 in the distributor instead of sending and parsing versions?
If we use the version field and store it at the distributor we need to reregister clients when they upgrade to a new connector version.

Reply:
I feel a little uneasy about the "spec" changing so willy nilly. Aren't you impacting tons of apps with this?